content/docs/join_us/recruitment_FAQs.md

```diff
@@ -241,6 +241,24 @@ This list is not exhaustive, but some examples include:
 - Research programming support (running [drop-in sessions]({{< relref "docs/regular_events/drop-in_sessions" >}}) for people at the Turing)
 - Knowledge sharing (organising our [tech talks]({{< relref "docs/regular_events/lunchtime_tech_talks.md" >}}))
 
+### Can you tell me more about what it is like working at The Alan Turing Institute?
+
+At The Turing, we want to make sure you feel comfortable and supported in how you work. Here’s a bit about what to expect:
+
+- **Hybrid Working**: We only ask that you come into the office four days a month — the rest is up to you! Some of the team work primarily remotely, while others come in every day. There is no pressure or expectation to be in the office more often than the minimum four days.
+
+- **Flexible Hours**: Our core hours are 10 am to 4 pm, but you’re free to work whenever works best for you. Whether you’re an early bird or prefer working later, we trust you to manage your time in a way that fits your life and responsibilities. We also understand that everyone has commitments — whether it’s a doctor’s appointment, picking up kids, or running errands — so taking time for those is completely fine.
+
+- **Work-Life Balance**: We value a healthy work-life balance and believe you should be able to switch off outside of work. There is no expectation to respond to emails or messages outside your working hours!
+
+- **Hot Desks**: We use hot desks, meaning you can sit wherever you like when you’re in the office, on a first-come, first-served basis. The Research Engineering Group typically sits near one another on the first floor. If you need a permanent or bespoke workspace (such as standing/adjustable desks, footrests, or ergonomic keyboards), we can work with the facilities team to make it happen.
+
+- **Come as You Are**: We don’t have a dress code, so you can dress in whatever makes you feel comfortable. Just keep it professional when needed!
```

Comment:
Although this again is true in my experience, I wonder if there's an implicit dress code that we just don't acknowledge. Would it make sense to link to a photo of the REG team to give a real impression of what people actually consider to be "normal"? (I can't think of many... there are some photos online of REG hackweek).

Reply:
This is very true, we do have an implicit casual dress code. 
Good idea. The away day picture is probably quite representative.